### PR TITLE
CRITICAL bug fix

### PR DESCRIPTION
**Pull request by webdev03**
Oops..

### CODE REVIEW

**Comment by webdev03** [comment]
@hello-smile6

**Comment by hello-smile6** [comment]
Really? Formatting???

**Comment by hello-smile6** [comment]
I need an auto-format workflow...

**Comment by webdev03** [comment]
Sorry about the formatting..

**Comment by hello-smile6** [comment]
> Sorry about the formatting..

Literally the exact same as the other 2 most recent versions. But I get it.

**Comment by webdev03** [comment]
I wrote "Critical bug" in the title!!

File: neofetch.js [hunk 1]
```diff
@@ -155,24 +155,24 @@ Neofetch.getData = function (opts) {
...
-          if (navigator?.userAgent?.includes("iPhone OS")) {
-            // They give the same logo so it doesn't matter
-            cfg.os = "Mac";
-          }
-          // iPadOS detection
-          if (navigator?.userAgent?.includes("iPad")) {
-            // They give the same logo so it doesn't matter
-            cfg.os = "Mac";
-          }
+        }
```
Comment (webdev03) [inline neofetch.js:158]: This part

Comment (hello-smile6) [reply neofetch.js:158]: Wait, that broke things?

**Comment by webdev03** [comment]
I think I am just too tired LOL

**Pull request merged**